### DISCUSSION TITLE
我的第一台组装主机

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#11] [General]
# 我的第一台组装主机

我的第一台组装主机 - 来自Xzonn的小站

https://xzonn.top/posts/DIY-My-Computer.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Sytus 评论于 2023-06-11 13:10:53 (UTC+8)*****

看起来很不错，性价比很高:)
